dat-schema/_Core.gql

```diff
@@ -89,7 +89,19 @@ type ActiveSkills {
   _: bool
 }
 
-enum ActiveSkillTargetTypes @indexing(first: 0) { _ }
+enum ActiveSkillTargetTypes @indexing(first: 1) {
+  TARGETABLE_GROUND
+  ENEMY
+  WALKABLE_GROUND
+  ANYWHERE_SELF_TARGET
+  ITEM
+  CORPSE
+  UNUSED1
```

Comment:
Do we want it to be `_` instead? (exported as `null`)

Reply:
Yea that might actually be better, ill change it